numcmctools/plotstack.py

```diff
@@ -97,7 +97,7 @@ def add_plot(self,variables, priors, empirical_priors, bins, axrange=None, mo_op
                 continue
             self.plotted_variables.append(var)
 
-        self.plots.append(Plot(variables, plot_jacobians, self.plotted_empirical_priors.keys(), bins, axrange, mo_option))
+        self.plots.append(Plot(variables, plot_jacobians, self.plotted_empirical_priors.keys(), bins, axrange, mo_option)) 
```

Comment:
While testing, I encountered another bug, small but critical for some applications.
Each plot should only have empirical priors that are added to that plot, not all the known empirical priors...
```suggestion
        self.plots.append(Plot(variables, plot_jacobians, empirical_priors, bins, axrange, mo_option)) 
```